test/unit/theory/theory_bv_opt_white.cpp

```diff
@@ -151,5 +151,31 @@ TEST_F(TestTheoryWhiteBVOpt, signed_max)
             << std::endl;
 }
 
+TEST_F(TestTheoryWhiteBVOpt, min_boundary)
+{
+  Node x = d_nodeManager->mkVar(*d_BV32Type);
+  // this variable is important on triggering the bug
```

Comment:
```suggestion
```
Having this comment here is confusing, because it is not clear what bug you are referring to.  Please remove the comment -- it is clear from lines 163--164 that this test was added due to a bugfix, and it is clear from line 165 that the variable `y` is important, since it is used there.

Reply:
... removed